tools/pylib/_boutcore_build/boutcore.pyx.in

```diff
@@ -104,14 +108,19 @@ cdef class Field3D:
         return f3d
     @classmethod
     def fromCollect(cls,name,tind=-1,mesh=None,ignoreDataType=False, **kwargs):
-        """fromCollect(cls,name,tind=-1,mesh=None,ignoreDataType=False, **kwargs)
-
+        """
         Create a Field3D from reading in a datafile via collect.
 
-        :param tind: time slice to read
-        :param mesh: if not defined, use global mesh
-        :param ignoreDataType: Do not fail if data is not float64
-        :param **kwargs: remaining arguments are passed to collect
+        Parameters
+        ----------
+        tind : int
+            time slice to read
+        mesh : Mesh
+            if not defined, use global mesh
+        ignoreDataType : bool
+            Do not fail if data is not float64
+         **kwargs :
```

Comment:
I think you've got an extra space at the start here that might prevent it rendering correctly. Also I don't think the `:` is needed if there's no type specified